docs/gettingstarted.rst

```diff
@@ -229,7 +229,59 @@ a string or integer. Below is an implementation of this design in the Python cod
   async def test(ctx, optone: str):
     await ctx.send(content=f"Wow, you actually chose {optone}? :(")
 
+Want to restrict access? Setup permissions!
+-------------------------------------------
+
+Slash commands also supports ability to set permissions to allow only certain roles and/or users 
+to run a slash command. Permissions can be applied to both global and guild based commands. They 
+are defined per guild, and each guild can have multiple permissions. Here table that shows the JSON 
+structure of how choices are represented for a single permission:
+
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| **Field**   | **Type**                                   | **Description**                                                                                     |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| id          | int                                        | Snowflake value of type specified. Represents the target to apply permissions on.                   |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| type        | int                                        | An `ApplicationCommandPermissionType`_.                                                             |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| permission  | boolean                                    | true to allow, false, to disallow.                                                                  |
```

Comment:
`true` to allow, `false` to disallow

This is per the Discord v8 docs to keep consistency.

Reply:
Just been following this..does the table need that much empty space?

Reply:
Yes due to RST formatting, that's how they have to be structured.

Reply:
Not necessarily see below suggestions.